data/bbike.sh

```diff
@@ -1,17 +0,0 @@
-#!/bin/bash
-# This downloads all the cities from https://download.bbbike.org/osm/bbbike/
-# and tries to convert them. Don't eat bbike's bandwidth, please.
```

Comment:
Did bbike.sh and mass_import.sh just go away? Old unused code?

Reply:
Yup, these were from https://github.com/a-b-street/docs/blob/ecce62e8bd59439d2d01e799e1dcba39418e71e6/book/src/tech/dev/mass_import.md, when I was trying to import a bunch of city centers for a demo to the OSM community. The scripts stopped working a while ago, and I don't have plans to attempt something like this again, so cleanup time